docs/en/04_Changelogs/4.0.0.md

```diff
@@ -393,6 +393,29 @@ all changed project files.
 This will resolve the majority of upgrading work, but for specific changes that will
 require manual intervention, please see the below upgrading notes.
 
+### Update theme setting
+
+SSViewer no longer takes a single theme - instead themes are set in a cascade. To upgrade from 3, 
+change:
+
+	theme: simple
```

Comment:
Actually, it is now. Fixed it after writing this. Will update. (The old behaviour supported it but didn't add `$default` so wasn't useful).